tests/components/ssdp/test_init.py

```diff
@@ -22,6 +22,17 @@
 from tests.common import async_fire_time_changed
 
 
+@pytest.fixture(autouse=True)
```

Comment:
Why are we changing scope of the fixture for ssdp?

Reply:
You told me before, in the comment from two days ago?

Reply:
I think I misunderstood you. Can you elaborate on the comment from two days ago?

Reply:
I asked that we not change the scope of any fixture unless there's a bug with the current scope. A fixture in the package `conftest` module will apply to all tests of that package. A fixture in a test module will only apply to the tests in that module. So moving a fixture is significant.